### DISCUSSION TITLE
Container is not running

### DISCUSSION THREAD

**Discussion by Smolevich** [community/community#26107] [Actions]
<p>I have a ci file</p>
<pre><code>name: CI

on: [push]

jobs:
  build:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        php: [5.6, 7.0, 7.1, 7.2, 7.3]
    container:
      image: php:${{ matrix.php }}-alpine
    steps:
    - uses: actions/checkout@v1
    - name: Show php version
      run: php -v
    - name: Install composer
      run: apk add wget &amp;&amp; wget -qO- https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer;
    - name: Install dependencies
      run: composer install --prefer-source
</code></pre>
<p>And on step <strong>Show php version </strong> error occured Error response from daemon: Container affade1e9552fd8565103588880dffc890da1c997494f05df43e7ec32c12b569 is not running<br><br>How I can solve this problem?</p>

**Comment by chrispat** [comment, marked as answer]
<p>We currently do not support Alpine containers for job level containers.  You will need a container that is capable of running the standard linux distribution of node.  We are working on changes to enable use of alpine based containers.</p>
<p>Update: 9/18/2019 Alpine containers are now supported at the job and service level <a href="https://github.blog/changelog/2019-09-18-improvements-to-github-actions/" rel="nofollow noopener">https://github.blog/changelog/2019-09-18-improvements-to-github-actions/</a></p>
